addons/l10n_in/tests/test_hsn_summary.py

```diff
@@ -578,3 +578,35 @@ def test_l10n_in_hsn_summary_4(self):
             },
         ))
         self._assert_tests(tests)
+
+    def test_l10n_in_hsn_summary_5(self):
+        """ Line with discount """
+        tests = []
+        base_lines1 = [
+            self.create_base_line_dict(self.test_hsn_code_1, 1.0, 1000.0, 50.0, self.uom_unit),
```

Comment:
as far as I can tell, this is not testing the change in the code, just testing the change in the test method "create_base_line_dict"

if you wanted to test the change in the code, you would need to call the method `_l10n_in_get_hsn_summary_table`